geanygendoc/src/ggd-tag-utils.c

```diff
@@ -274,7 +267,7 @@ static const struct {
   { tm_tag_externvar_t,       "extern"    },
   { tm_tag_macro_t,           "define"    },
   { tm_tag_macro_with_arg_t,  "macro"     },
-  { tm_tag_file_t,            "file"      }
+  { tm_tag_local_var_t,       "local variable" }
```

Comment:
Just noticed that this change depends on the new API with renamed `tm_tag_file_t` to `tm_tag_local_var_t` so we should require that API version (and also bump API version in Geany which we didn't do).

Alternatively, and this might be a better solution, we could just remove this line. If I understand correctly the purpose of this plugin, it serves for generating documentation stubs for symbols at the top level and local variables aren't interesting for it. I checked if something in the plugin doesn't rely on the exact size of this array but this array is only accessed from `ggd_tag_type_get_name()` and `ggd_tag_type_from_name()` below and they use the real size of the array.

Reply:
If its not used then there seems no point in leaving it in the array and then the plugin is API agnostic.

Reply:
`tm_tag_file_t` isn't used but `tm_tag_local_var_t` to which we renamed it is now. But I think `tm_tag_local_var_t` isn't interesting for the plugin so it doesn't have to be listed here (so we don't have to depend on a new Geany API version).

Reply:
Agree, I meant "not used by the plugin".